gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Sun Dec 18 11:08:04 CET 2016
+#Thu Sep 21 16:25:16 CEST 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.2.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.5-bin.zip
```

Comment:
latest and greatest is 4.2 - does it work with it as well?

Reply:
xtext still uses 3.5 cause of tons of gradle 4x bugs